sources/kube.go

```diff
@@ -36,14 +37,66 @@ var kubeVersions = []string{"v0.3"}
 // Cadvisor port in kubernetes.
 const cadvisorPort = 4194
 
+type PodInstance struct {
+	Pod    string
+	HostIp string
```

Comment:
The errors are probably due to problem from host. Do you want to add UID in addition to pod and host name?